contrib/roles/linux/kubernetes/tasks/prepare_minion.yml

```diff
@@ -85,7 +85,7 @@
 
 - name: Kubernetes Minion | Prepare OVN certs
   shell: |
-    ovs-vsctl set Open_vSwitch . \
+    ovs-vsctl --timeout={{TIMEOUT}} set Open_vSwitch . \
       external_ids:k8s-api-server="http://{{ host_public_ip }}:8080" \
       external_ids:k8s-api-token="{{TOKEN}}"
```

Comment:
There should be ``&&`` at the end of this line to handle the error for the command.

Reply:
Should ideally be a script then. Its two disparate commands.